src/Raven.Client/Util/JavascriptConversionExtensions.cs

```diff
@@ -1008,6 +1008,185 @@ public override void ConvertToJavascript(JavascriptConversionContext context)
+            {
+                _alias = inputAlias;
+            }
+
+            public override void ConvertToJavascript(JavascriptConversionContext context)
+            {
+                var methodCallExpression = context.Node as MethodCallExpression;
+
+                if (methodCallExpression?.Method.Name != nameof(RavenQuery.Include))
+                    return;
+
+                if (methodCallExpression.Method.DeclaringType != typeof(RavenQuery))
+                    return;
+
+                if (methodCallExpression.Arguments.Count != 1)
+                    return;
+
+                context.PreventDefault();
+                
+                var writer = context.GetWriter();
+
+                //if not a lambda expression then
+                if (!methodCallExpression.Arguments[0].Type.BaseType.Name.Contains("LambdaExpression"))
+                {
+                    using (writer.Operation(methodCallExpression))
+                    {
+                        var arg = methodCallExpression.Arguments[0];
+
+                        if (arg is ConstantExpression { Value: string target1})
+                        {
+                            writer.Write("include(");
+                            writer.Write(target1);
+                            writer.Write(")");
+                            return;
+                        }
+                        throw new NotSupportedException("You can't use the include that way try: let _ = RavenQuery.Include<T>(Expression<Func<T, string>> path)");
+                    }
+                }
+
+                //if got here then it's lambda expression then call alias and 
+                using (writer.Operation(methodCallExpression))
+                {
+                    var arg = methodCallExpression.Arguments[0];
+
+                    if (arg is UnaryExpression { Operand: LambdaExpression { Body: BinaryExpression { Left: MethodCallExpression mce } } })
+                    {
+                        var objReplace = (MemberExpression)mce.Object;
+                        var replaceExprassion = Expression.Parameter(objReplace.Expression.Type, _alias);
+                        var objReplace1 = objReplace.Update(replaceExprassion);
+                        var curmce = mce.Update(objReplace1, mce.Arguments);
+
+                        writer.Write("include(");
+                        context.Visitor.Visit(curmce);
+                        writer.Write(")");
+                        return;
+                    }
+
+                    if (arg is UnaryExpression { Operand: LambdaExpression arg2 })
+                    {
+                        var targetExpression = (MemberExpression)arg2.Body;
+                        var replaceExprassion = Expression.Parameter(targetExpression.Expression.Type, _alias);
+                        targetExpression = targetExpression.Update(replaceExprassion);
+
+                        writer.Write("include(");
```

Comment:
this is legacy syntax as clearly is stated in `ScriptRunner`. Please use `includes.document`

Reply:
done